infra/src/main/kotlin/org/yapp/infra/aop/aspect/ControllerLoggingAspect.kt

```
@@ -0,0 +1,69 @@
+        if (!properties.controller.enabled) {
+            return joinPoint.proceed()
+        }
+
+        val startTime = Instant.now()
+        logRequestStart(joinPoint, startTime)
+
+        try {
+            val result = joinPoint.proceed()
+            logRequestSuccess(startTime)
+            return result
+        } catch (e: Throwable) {
```

detekt (1.23.8)

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)